benefits/enrollment/views.py

```diff
@@ -282,14 +281,13 @@ def success(request):
     request.path = "/enrollment/success"
     session.update(request, origin=reverse(ROUTE_SUCCESS))
 
-    eligibility = session.eligibility(request)
     flow = session.flow(request)
 
     if session.logged_in(request) and flow.claims_provider.supports_sign_out:
         # overwrite origin for a logged in user
         # if they click the logout button, they are taken to the new route
         session.update(request, origin=reverse(ROUTE_LOGGED_OUT))
 
-    analytics.returned_success(request, eligibility.group_id)
+    analytics.returned_success(request, flow.group_id)
```

Comment:
:+1: This is the pattern we should be embracing everywhere: using a `flow` object directly.

Reply:
Absolutely right, I made all the changes that were required in [`eligibility/views`](https://github.com/cal-itp/benefits/blob/903f76f7bf932cf401dba9e78887ee6b2da5c1b6/benefits/eligibility/views.py)